.github/workflows/acctest.yml

```diff
@@ -68,10 +68,25 @@ jobs:
 
         METAL_AUTH_TOKEN: ${{ secrets.PACKET_AUTH_TOKEN }}
       run: |
-        go test -v -cover -parallel 4 -timeout 120m ./metal
+        go test -v -json -coverprofile coverage.txt -covermode=atomic -parallel 4 -timeout 180m ./metal > test.json
+    - name: Annotate tests
+      if: always()
+      uses: guyarb/golang-test-annotations@v0.4.0
+      with:
+        test-results: test.json
     - name: Sweeper
       if: ${{ always() }}
       env:
         METAL_AUTH_TOKEN: ${{ secrets.PACKET_AUTH_TOKEN }}
       run: |
-        go test ./metal -v -sweep="tf_test"
+        go test ./metal -v -json -sweep="tf_test"  > sweep.json
+    - name: Annotate sweep
+      if: always()
```

Comment:
This is a different format than the existing `${{ always() }}`.  I assume they are both acceptable.